Flutter/globalization.md

```diff
@@ -169,8 +169,8 @@ To use the [`Syncfusion Localization`](https://pub.dev/packages/syncfusion_local
 
 {% highlight dart %} 
 
-    import 'package:syncfusion_flutter_calendar/calendar.dart';
-    import 'package:syncfusion_localizations/syncfusion_localizations.dart';
+import 'package:syncfusion_flutter_calendar/calendar.dart';\
```

Comment:
@LokeshPalani Why you are added \ here. Please remove it.

Reply:
Hi @LavanyaGowtham2021 ,

I have removed the \ symbol .
Please check below commit.
Link - https://github.com/syncfusion-content/flutter-docs/pull/408/commits/d652628459b47f3bb9ee8f6c222e7c53556f22dc

Regards,
Lokesh Palani.